### PR TITLE
Improved Documentation: Update PULL_REQUEST_TEMPLATE.md

### LINKED ISSUE #301: Improve PR template
Will be adding some more things

### PR DESCRIPTION
**Pull request by saarthakchats**
### When you change [things like this in brackets] please remember to delete brackets after you change it.
  
 -# This pull request is related to issue(s), 
 Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.
 
  [x] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
 - [ ] Included a Preview link and screenshot showning after and before the changes.
 - [x] Included a description of change below.
 
 
 # Changes done in this Pull Request
Have developed PR template. Added more detailed explainations and options.
 
  
   ---- ---- None

  ### Screenshots,
  [If any]

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,7 +1,18 @@
 ## Don't delete anything without explicit instructions from a maintainer.
 ### When you change [things like this in brackets] please remember to delete brackets after you change it.
 
-# This pull request is related to issue(s), 
+Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.
+
+- [ ] Read and understood (see CONTRIBUTING.md)
```
Comment (niccokunzmann) [inline .github/PULL_REQUEST_TEMPLATE.md]: Here, you can add a link to the file. By the way: github shows this at the top.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
+- [ ] Included a Preview link and screenshot showning after and before the changes.
+- [ ] Included a description of change below.
+- [ ] Squashed the commits.
+
+# Changes done in this Pull Request
+
+- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
```
Comment (niccokunzmann) [inline .github/PULL_REQUEST_TEMPLATE.md]: could also be raw git. Please add a link so they can read more about it.

**Comment by saarthakchats** [comment]
@niccokunzmann Can we merge now 👍

**Comment by niccokunzmann** [comment]
Thanks for the pr and the review!

**Pull request merged**